lib/easypost/models/smartrate.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# The Smartrate object contains details about the smartrate requests.
+class EasyPost::Models::Smartrate < EasyPost::Models::EasyPostObject
```

Comment:
```suggestion
class EasyPost::Models::SmartRate < EasyPost::Models::EasyPostObject
```

issue: We should be using the proper names of our object where possible (when it doesn't break code such as snake casing because of it.

Please adjust throughout.

Reply:
Fixed